.editorconfig

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Would this still work under python/ because that's where Python-related scripts would be run from?